bitex/api/base.py

```diff
@@ -44,6 +44,7 @@ def __init__(self, *, addr, key, secret, version, config):
         self.key = key if key else None
         self.secret = secret if secret else None
         self.version = version if version else None
+        self.proxies = proxies if proxies else None
```

Comment:
This can be `proxies or None` - also for the already existing assignments of secret, key and version.

Reply:
not understand...

Reply:
Rewrite to
```
self.key = key or None
self.secret = secret or None
self.version = version or None
self.proxies = proxies or None
```